plugins/storage/volume/solidfire/src/org/apache/cloudstack/storage/datastore/provider/SolidFireHostListener.java

```diff
@@ -91,18 +280,5 @@ public boolean hostConnect(long hostId, long storagePoolId) {
         assert (answer instanceof ModifyStoragePoolAnswer) : "ModifyStoragePoolAnswer expected ; Pool = " + storagePool.getId() + " Host = " + hostId;
```

Comment:
Some like they; some don't. I'd say feel free to put asserts in your code. I don't think the community has a formal policy on it.